packages/core/src/tools/errorFilter.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { Configuration } from '../domain/configuration'
+import { mockClock } from '../../../core/test/specHelper'
+import { RawError } from './error'
+import { createErrorFilter, ErrorFilter } from './errorFilter'
+import { RelativeTime, relativeToClocks, resetNavigationStart } from './timeUtils'
+import { noop, ONE_MINUTE } from './utils'
+
+const CONFIGURATION = { maxErrorsByMinute: 1 } as Configuration
+
+describe('errorFilter', () => {
+  let errorFilter: ErrorFilter | undefined
+  let clock: jasmine.Clock
+  let cleanupClock: () => void
+
+  beforeEach(() => {
+    ;({ clock, stop: cleanupClock } = mockClock())
+    resetNavigationStart()
+  })
+
+  afterEach(() => {
+    cleanupClock()
+  })
+
+  it('allows to send an error', () => {
+    errorFilter = createErrorFilter(CONFIGURATION, noop)
+
+    expect(errorFilter.isLimitReached()).toBe(false)
+  })
+
+  it('prevents from sending an error when the limit is reached', () => {
+    errorFilter = createErrorFilter(CONFIGURATION, noop)
+
+    errorFilter.isLimitReached()
+    expect(errorFilter.isLimitReached()).toBe(true)
+  })
+
+  it('allows to send errors again once one minute have passed', () => {
+    errorFilter = createErrorFilter(CONFIGURATION, noop)
+
+    errorFilter.isLimitReached()
+    errorFilter.isLimitReached()
+    clock.tick(ONE_MINUTE)
+    expect(errorFilter.isLimitReached()).toBe(false)
+  })
+
+  it('calls the "onLimitReached" callback with the raw "limit" error to send when the limit is reached', () => {
```

Comment:
```suggestion
  it('calls the "onLimitReached" callback with the raw "limit reached" error to send when the limit is reached', () => {
```
and other occurrences to increase naming consistency